imports/utils.coffee

```diff
@@ -288,6 +288,7 @@ export createIncidentReportsFromEnhancements = (enhancements, options) ->
         timeAnnotation.timeRange.end
       ).endOf('day')
       if timeAnnotation.beginMoment > timeAnnotation.endMoment
+        console.log(timeAnnotation)
```

Comment:
I want to keep this log since it helps to track down the cause of "End date occurs before start date." errors.